source/documentation.blade.php

```diff
@@ -0,0 +1,48 @@
+@extends('_layouts.main')
+
+@section('body')
+    <!-- ====== Banner Start ====== -->
+    <section class="ud-page-banner">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-12">
+            <div class="ud-banner-content">
+              <h1>{{ $page->t( "Documentation") }}</h1>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- ====== Banner End ====== -->
+
+    <section class="ud-blog-grids">
+        <div class="container">
+          <section class="ud-404">
+            <div class="container">
+              <div class="row">
+                <div class="col-lg-12">
+                  <div class="ud-404-wrapper">
+                    <div class="ud-404-content">
+                      <h2 class="ud-404-title">{{ $page->t( "This documents the upcoming version of Libresign") }}</h2>
+                      <ul class="ud-404-links">
+                        <li>
+                          <a href="{{ $page->baseUrl }}documentation/user/instalation">{{ $page->t( "User") }}</a>
```

Comment:
Maybe only docs at all places instead of documentation. Is shorter. 

Would be good send to a home page of this collections because we also need to have a home at developer documentation.

I think that docs/user at Jigsaw side, will be a page outside of collection structure because the collection create the sub-folders and not the root folder of docs/user.
```suggestion
                          <a href="{{ $page->baseUrl }}docs/user">{{ $page->t( "User") }}</a>
```